src/commands/run.py

```diff
@@ -20,21 +20,31 @@ def add_commands(appliance):
     def run():
         pass
 
+    run_options = {
```

Comment:
Can this copy how `options` are shared in `status`:
https://github.com/alces-software/adminware/blob/8749db2a28195433edd892c21cc457f2aa79f0a6/src/commands/status.py#L17

Basically include the top level `option` key in the `run_options` hash

Reply:
done here https://github.com/alces-software/adminware/pull/166/commits/94dbd76ada46acd682b7008d9bc4962ec7122a9e